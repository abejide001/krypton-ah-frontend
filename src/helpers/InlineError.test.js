import React from 'react';
import { shallow } from 'enzyme';
import InlineError from './InlineError';

describe('SignIn Form', () => {
  test('should matches the snapshot', () => {
    const wrapper = shallow(<InlineError />);
    expect(wrapper).toMatchSnapshot();
  });
});
