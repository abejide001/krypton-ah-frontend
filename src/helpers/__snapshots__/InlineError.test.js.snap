// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn Form should matches the snapshot 1`] = `
<span
  className="inlineError"
/>
`;
